README.md

```diff
@@ -16,355 +16,16 @@ Kopytko Framework is part of Kopytko Roku Ecosystem which consists of:
-
-`componentsMapping` is handy to render mocked instead of real components to make test case more hermetic.
-For example, imagine you have a "BrowseView" Kopytko component which renders a lot of "CustomButton" Kopytko components.
-"CustomButton" has a lot of business logic code in its `componentDidMount` lifecycle method. Before every test case
-of the BrowseView component, `initKopytko` has to be called which also initializes its CustomButton components
-and unnecessarily runs their business logic. To avoid that, the CustomButtonMock component could be created (we recommend
-placing it in the `_mocks` folder, next to the original component) which extends just basic SceneGraph's `Group` component.
-To inform Kopytko framework to use CustomButtonMock instead of CustomButton, you can map it in `componentsMapping`:
-`initKopytko({}, { CustomButton: "CustomButtonMock" })`.
-
-### KopytkoFrameworkTestSuite
-Use `KopytkoFrameworkTestSuite` instead of basic Kopytko Unit Testing Framework's `KopytkoTestSuite`.
-It will automatically clear your component state after every test case and make your unit tests independent.
-
-### Checking element state after a rerender
-Currently, there's no way to test asynchronous code in our test framework, and since changes that cause a rerender
-are always done asynchronously, you can force a synchronous rerender using the `forceUpdate` method, so when
-you need to check if an element has some expected value after a rerender you can: change the desired prop,
-call `forceUpdate()` to cause the rerender, and then check the element to see if it has the expected values.
-
-### Using onKeyEvent() to fake remote key presses
-When you need to assert something after some key was pressed, you can call directly the `onKeyEvent(press, key)` method,
-which will call the method/execute the code related to the parameters you pass to it. Keep in mind that sometimes
-this is not enough to fake a key press, take a button selected event as an example, when you press the OK key while
-focusing a button the handler that handles this OK key press is set in the native Button component,
-not in your component, so in such cases you should test the callback function directly (`_onButtonSelected()`, for example).
+## Modules
+
+- Renderer: main module, inspired by JS React library, rendering components. Full documentation available in
+  [docs/renderer.md](docs/renderer.md)
+- Router: enables building an app with multiple views and allows navigation between them
+- Cache: a mechanism to store expirable external data
+- Event Bus: implementation of Event Bus simplifying the communication between components
+- HTTP: easy way to send HTTP requests
+- Modal: global UI component
+- Registry: a facade over native Roku's registry
+- Store: a mechanism to store reusable data
+- Theme: manages UI configuration and allows easy use in any place
```

Comment:
Please check if it's fine with you, guys

Reply:
We need to describe them in more details with code examples I think.